### PR TITLE
US74173: Update ECS deploy file to allow for Fargate deployment

### PR DESCRIPTION
**Pull request by murphry**

### CODE REVIEW

**Review by davidjameshowell — APPROVED** [review]
LGTM. Only note is that this is **not** backwards compatible with EC2 deployments of ECS services.

**Review by nickdemarco89 — APPROVED** [review]

**Review by davidjameshowell — APPROVED** [review]

File: Dockerfile [hunk 1]
```diff
@@ -1,13 +1,8 @@
...
-RUN apt-get update -y \
-    && apt-get install -y \
-        curl \
-        python-setuptools \
-        jq \
-    && easy_install pip \
-    && pip install awscli
+# Install packges needed
+RUN apk --no-cache add ca-certificates curl bash jq py3-pip 
+# RUN    pip install awscli
```
Comment (akaustav) [inline Dockerfile:5]: Consider removing this commented line if it is not necessary.

```suggestion
```

**Review by akaustav — APPROVED** [review]
One comment.

**Review by davidjameshowell — CHANGES_REQUESTED** [review]
Just saw, we still need the AWS CLI command still @murphry

File: Dockerfile [hunk 2]
```diff
@@ -1,13 +1,8 @@
...
-RUN apt-get update -y \
-    && apt-get install -y \
-        curl \
-        python-setuptools \
-        jq \
-    && easy_install pip \
-    && pip install awscli
+# Install packges needed
+RUN apk --no-cache add ca-certificates curl bash jq py3-pip 
+RUN    pip install awscli
```
Comment (KHannel) [inline Dockerfile:5]: should these be combined into a single RUN?

**Pull request merged**